src/test/java/nearsoft/academy/bigdata/recommendation/MovieRecommender.java

```diff
@@ -0,0 +1,104 @@
+package nearsoft.academy.bigdata.recommendation;
+
+import org.apache.commons.collections.functors.WhileClosure;
+import org.apache.commons.math.stat.descriptive.summary.Product;
+
+import java.io.BufferedReader;
+import java.util.*;
+import java.io.FileReader;
+
+public class MovieRecommender {
+
+
+    ArrayList<Review> listaDeReviews = new ArrayList<>();
+    int contadorDeReviews = 0;
+    public MovieRecommender(String filepath ) {
+
+       try{
+            FileReader fr = new FileReader(filepath);
+            BufferedReader br = new BufferedReader(fr);
+
+            String nuevaLinea;
+
+            Review objeto = new Review();
+            while ((nuevaLinea = br.readLine()) != null) {
+
+                String words[] = nuevaLinea.split(" ");
```

Comment:
Why are we using blank space as separator character? a line could have more than one blanck space and we will not get all the values with words[1], as the array returned by split will have more elements